src/lib/utils/getRefinements.ts

```diff
@@ -47,18 +46,22 @@ function getRefinement(
   type: Refinement['type'],
   attributeName: Refinement['attributeName'],
   name: Refinement['name'],
-  resultsFacets: SearchResults['facets' | 'hierarchicalFacets']
+  resultsFacets: SearchResults['facets' | 'hierarchicalFacets'] = []
 ): Refinement {
   const res: Refinement = { type, attributeName, name };
-  let facet: any = find(resultsFacets, { name: attributeName });
+  let facet: any = (resultsFacets as Array<{ name: string }>).find(
```

Comment:
`resultsFacets` is not correctly inferred?

Reply:
No, it seems like TypeScript cannot infer it with the syntax:


```ts
SearchResults['facets' | 'hierarchicalFacets']
```

Reply:
We should probably inline the value then.